release_notes/Service_Fabric_ReleaseNotes_91.md

```diff
@@ -0,0 +1,121 @@
+# Microsoft Azure Service Fabric 9.1 Release Notes
+
+* [Service Fabric Packages and Versions](#service-fabric-packages-and-versions)
+* [Current Breaking Changes](#current-breaking-changes)
+* [Key Announcements](#key-announcements)
+* [Service Fabric Feature and Bug Fixes](#service-fabric-feature-and-bug-fixes)
+* [Retirement and Deprecation Path Callouts](#retirement-and-deprecation-path-callouts)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+## Service Fabric Packages and Versions
+The following packages and versions are part of this release:
+
+| Service | Platform | Version |
+|---------|----------|---------|
+|Service Fabric Runtime| Ubuntu 18 <br> Ubuntu 20 <br> Windows | 9.1.1206.1 <br> 9.1.1206.1 <br> 9.1.1390.9590 |
+|Service Fabric for Windows Server|Service Fabric Standalone Installer Package | 9.1.1390.9590 |
+|.NET SDK |Windows .NET SDK <br> Microsoft.ServiceFabric <br> Reliable Services and Reliable Actors <br> ASP.NET Core Service Fabric integration| 6.1.1390  <br> 9.1.1390 <br> 9.1.1390 <br> 9.1.1390 |
+|Java SDK  |Java for Linux SDK  | 1.0.6 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.1 |
+
+## Current Breaking Changes
+
+**Breaking Changes with  BackupRestoreService:**
+If a Service Fabric cluster has periodic backup enabled on any of the app/service/partition, post upgrade to 9.1.1390.9590, BRS will fail to deserialize old BackupMetadata. BRS will also stop taking backup and restore on the partition/service/app in question with changes in the new release even though the user app, cluster, and BRS shows healthy
+
+**Identifying the issue:**
+There are two ways to identify and confirm the issue
+
+A. If periodic backups were happening on any partition, it should be visible on SFX under Cluster->Application->Service->Partition->Backup. Here list of all backups being taken with creation time is available. Using this info and upgrade time, customer can identify whether a backup policy was enabled, backups were happening before upgrade and whether backups are happening post upgrade.
```

Comment:
Instead of --
"SFX under Cluster->Application->Service->Partition->Backup" 
Use --
"Service Fabric Explorer (SFX) under Cluster -> Application -> Service -> Partition -> Backup"